statsmodels/genmod/tests/test_gee.py

```diff
@@ -1356,8 +1381,12 @@ def test_wrapper(self):
 
         check_wrapper(rslt2)
 
+@dec.skipif(not have_matplotlib)
 def test_plots():
 
+    if not have_matplotlib:
+        raise nose.SkipTest('No tests here')
+
```

Comment:
this looks like a duplicate skip, the `dec.skipif` is doing the same already. 
IIRC, we use `raise nose.SkipTest` in a test class that we want to completely skip, for individual methods the decorator is more convenient.